test/components/preprocessors/test_sentence_tokenizer.py

```diff
@@ -98,3 +99,16 @@ def test_quote_spans_regex():
     text5 = "This text has no quotes."
     matches5 = list(QUOTE_SPANS_RE.finditer(text5))
     assert len(matches5) == 0
+
+
+def test_split_sentences_performance() -> None:
+    # make sure our regex is not vulnerable to Regex Denial of Service (ReDoS)
+    # https://owasp.org/www-community/attacks/Regular_expression_Denial_of_Service_-_ReDoS
+    # this is a very long string, roughly 500 MB, but it should not take more than 5 seconds to process
+    splitter = SentenceSplitter()
+    text = " " + '"' * 20 + "A" * 500000000 + "B"
```

Comment:
do you think that it would make sense to reduce the string length (maybe use 50000000 instead of 500000000)
and the time, in order to both verify the behavior and keep the CI fast?

Reply:
yes, makes sense 👍🏽